### PR TITLE
[IMP] mail: remove isDisabled props for FollowerListMenu component

### PR DESCRIPTION
**Pull request by phenix-factory**
Part of task-2693304

### CODE REVIEW

**Review by alexkuhn — APPROVED** [review]
@robodoo r+

**Comment by alexkuhn** [comment]
@robodoo r-

**Comment by phenix-factory** [comment]
@alexkuhn This should be ready for review now.

File: addons/mail/static/src/models/follower_list_menu_view.js [hunk 1]
```diff
@@ -34,12 +34,22 @@ registerModel({
                     break;
             }
         },
+        /**
+         * @private
+         * @returns {Boolean}
+         */
+        _computeIsDisabled() {
+            return Boolean(!this.chatterOwner || this.chatterOwner.isDisabled);
```
Comment (seb-odoo) [inline addons/mail/static/src/models/follower_list_menu_view.js:42]: `!this.chatterOwner` should not be necessary?

File: addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js [hunk 2]
```diff
@@ -1,26 +1,67 @@
 /** @odoo-module **/
 
-import { insert, replace } from '@mail/model/model_field_command';
+import { makeDeferred } from '@mail/utils/deferred';
 import {
+    afterNextRender,
     start,
     startServer,
 } from '@mail/../tests/helpers/test_utils';
+import FormView from 'web.FormView';
```
Comment (seb-odoo) [inline addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js:9]: guideline: new line between imports from different modules

File: addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js [hunk 3]
```diff
@@ -1,26 +1,67 @@
...
 
 import Bus from 'web.Bus';
 
 QUnit.module('mail', {}, function () {
 QUnit.module('components', {}, function () {
-QUnit.module('follower_list_menu_tests.js');
+QUnit.module('follower_list_menu_tests.js', {
```
Comment (seb-odoo) [inline addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js:15]: cc @tsm-odoo

File: addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js [hunk 4]
```diff
@@ -106,37 +159,54 @@ QUnit.test('click on "add followers" button', async function (assert) {
...
-                "The 'add followers' action should be a wizard invite of mail module"
-            );
-            assert.strictEqual(
-                action.type,
-                "ir.actions.act_window",
-                "The 'add followers' action should be of type 'ir.actions.act_window'"
-            );
-            pyEnv['res.partner'].write([action.context.default_res_id], { message_follower_ids: [mailFollowerId1] });
-            options.on_close();
+        assert.step('action:open_view');
```
Comment (seb-odoo) [inline addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js:162]: don't change indent here, it was done on purpose in preparation of following changes

**Review by seb-odoo — CHANGES_REQUESTED** [review]
see comments above

File: addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js (hunk 4 above)
Comment (phenix-factory) [reply addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js]: Ho ok !

**Comment by phenix-factory** [comment]
Comments done !

**Comment by seb-odoo** [comment]
runbot's red

File: addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js (hunk 2 above)
Comment (seb-odoo) [reply addons/mail/static/tests/qunit_suite_tests/components/follower_list_menu_tests.js:9]: comment not done?

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**